src/pages/dashboard/sentry/NewSentry.js

```diff
@@ -157,22 +161,78 @@ class NewSentry extends Component {
     }
   }
 
+  async handleServerChanSubmit(e) {
+    this.setState({
+      addLoading: true,
```

Comment:
plz reuse code instead of copy paste. Actually adding a new notifiction method when creating a sentry is not an urgent feature. If you think it's a bit complicated, maybe considering remove this and add it back later.